radixdlt-core/radixdlt/src/test/java/org/radix/serialization/SerializeMessageObject.java

```diff
@@ -64,13 +64,15 @@
 
 package org.radix.serialization;
 
+import com.radixdlt.network.messaging.Message;
+
 import java.util.function.Supplier;
 
 /**
  * Raft of tests for serialization of objects.
  * <p>
  * This class extends {@link SerializeObject} to set up databases
- * required by most classes that extend {@link org.radix.network.messaging.Message}.
+ * required by most classes that extend {@link Message}.
  *
```

Comment:
This change might cause "unused import" false positive (don't remember which one complains - checkstyle or Sonar)